stack.h

```diff
@@ -60,7 +68,7 @@ template <typename T> class Stack {
     }
 
     bool empty () {
-        return current_index == 0;
+        return (current_index == 0);
```

Comment:
This is not an assignment so parentheses are not required

Reply:
Yeah, not requaired but i don't know how for you, but for me that's more readible